nrf-softdevice/Cargo.toml

```diff
@@ -32,6 +32,8 @@ ble-sec = []
 
 critical-section-impl = ["critical-section/restore-state-bool"]
 
+usable-from-interrupts = [ "critical-section-impl" ]
```

Comment:
it should not enable `critical-section-impl`, in case the user wants another impl.

(It is fine if the build breaks if the user enables `usable-from-interrupts` but doesn't enable any CS impl from any crate)

Reply:
on it